clients/spark/core/src/main/scala/io/treeverse/clients/Exporter.scala

```diff
@@ -104,7 +104,7 @@ class Exporter(spark : SparkSession, apiClient: ApiClient, filter: KeyFilter, re
 
   private def writeSummaryFile(success: Boolean, commitID: String, content : String) = {
     val suffix = if(success) "SUCCESS" else "FAILURE"
-    val time = ZonedDateTime.now(ZoneOffset.UTC).format(DateTimeFormatter.ISO_INSTANT)
+    val time = DateTimeFormatter.ISO_INSTANT.format(ZonedDateTime.now())
```

Comment:
StillLGTM, but still wary of this code being "zoned" (to coin a word).  Just get the current instant, even with [`java.time.Clock.systemUTC`](https://docs.oracle.com/javase/8/docs/api/java/time/Clock.html#systemUTC--)`.instant()`.

Timezones have no place in code, *except* when formatting a local time for human consumption or arithmetic with "civil" durations (years, months, and often days).